MapboxNavigation/NavigationViewController.swift

```diff
@@ -373,6 +399,26 @@ public class NavigationViewController: NavigationPulleyViewController, RouteMapV
         if routeProgress.currentLegProgress.alertUserLevel == .arrive {
             navigationDelegate?.navigationViewController?(self, didArriveAt: destination)
         }
+        
+        applyStyle()
```

Comment:
Are you fearful of constantly updating the style? If so, i added this guard here https://github.com/mapbox/mapbox-navigation-ios/pull/519/commits/e52c62138b8c8b480bb949e1d52893199b03e66d